pom.xml

```diff
@@ -82,6 +82,9 @@
 		<license.copyrightOwners>Fiji developers.</license.copyrightOwners>
 		<license.projectName>Fiji distribution of ImageJ for the life sciences.</license.projectName>
 
+		<!-- https://github.com/scijava/scijava-coding-style -->
+		<scijava.coding-style>imglib2</scijava.coding-style>
```

Comment:
What this line does? Is it black magic?

Reply:
That's a recent addition by @maarzt to the SciJava ecosystem, shipping with the latest `pom-scijava`.

You can run:

```
mvn formatter:format
```

and

```
mvn impsort:sort
```

to get your code formatted and imports sorted, in the style of your choice.
See the link in the code.